### DISCUSSION TITLE
Cannot import slimSelect css in app.css with vite

### DISCUSSION THREAD

**Discussion by kolydart** [Power-Components/livewire-powergrid#982] [Q&A]
In a laravel 10 / livewire-powergrid 4.4 installation, I am trying to enable slimSelect. 

I have followed the [instructions](https://v4.livewire-powergrid.com/table/column-filters.html#filter-multiselect), but when I run `npm run dev`, I get the error _Unable to resolve `@import "~slim-select/dist/slimselect.css"` from /resources/css_ . 

There is something wrong with adding `@import "~slim-select/dist/slimselect.css";` in resources/css/app.css but I cant' tell what.

This is a diff patch of the changes from a working installation:
[diff.patch](https://github.com/Power-Components/livewire-powergrid/files/11504974/diff.patch)

Any help would be greatly appreciated.

**Comment by lukasx102** [comment]
I have some problem with TOMselect and vite
can not import to CSS
@import "~tom-select/dist/scss/tom-select.bootstrap5";
1.issue can not resolve file tom-select.bootstrap5" so I added .scss to load it 
@import "~tom-select/dist/scss/tom-select.bootstrap5.scss";
2. issue 
 [vite] Internal server error: [postcss] ENOENT: no such file or directory, stat '/var/www/html/~tom-select/dist/scss/tom-select.bootstrap5.scss'
  Plugin: vite:css
  File: /var/www/html/resources/css/app.css

What  am I doing wrong? 

thanks

**Comment by kolydart** [comment]
In [powergrid-demo app.css file](https://github.com/Power-Components/powergrid-demo/blob/main/resources/css/app.css), I notice that the import is formed like `@import "./../../node_modules/slim-select/dist/slimselect.css"`.
I used this notation and `npm run build` completed successfully.
So, I guess case is closed.
